app/src/org/odk/collect/android/widgets/IntentWidget.java

```diff
@@ -104,9 +98,6 @@ public void makeTextView() {
     public void makeButton(){
         setOrientation(LinearLayout.VERTICAL);
 
-        TableLayout.LayoutParams params = new TableLayout.LayoutParams();
-        params.setMargins(7, 5, 7, 5);
-
```

Comment:
`params` is never passed into anything, so it doesn't seem used, but the linter doesn't say it is dead code. I just tested it and it is okay to remove.